Alignment/MillePedeAlignmentAlgorithm/plugins/MillePedeDQMModule.h

```diff
@@ -65,18 +61,20 @@ class MillePedeDQMModule : public DQMEDHarvester {
     void fillExpertHistos();
 
     void fillExpertHisto(MonitorElement* histo,
-                         const double cut,
-                         const double sigCut,
-                         const double maxMoveCut,
-                         const double maxErrorCut,
+			 std::array<double, 6> cut, 
+			 std::array<double, 6> sigCut, 
+			 std::array<double, 6> maxMoveCut, 
+			 std::array<double, 6> maxErrorCut,
                          std::array<double, 6> obs,
                          std::array<double, 6> obsErr);
 
     bool setupChanged(const edm::EventSetup&);
+    int getIndexFromString(const std::string& alignableId);
 
-  //========================== PRIVATE DATA ====================================
-  //============================================================================
+    //========================== PRIVATE DATA ====================================
+    //============================================================================
 
+    const AlignPCLThresholds* theThresholds; 
```

Comment:
@mmusich this could also be a `shared_ptr` pointing to the same object as the `MillePedeFileReader` does, right?
This way you don't have to delete it in the end.
This again leads to the question on the ownership of objects pointed to by `edm::Handle::product`.
From your code it looks like the ownership is transferred because you delete it in the end.
Looks really like we should replace the raw pointers simply with smart pointers here.